### DISCUSSION TITLE
Can I ignore "description" when using Taskwarrior integration?

### DISCUSSION THREAD

**Discussion by robertsawko** [GothenburgBitFactory/timewarrior#569] [Q&A]
Using  `task start/stop/done ...`, I'd like `timew` to only assign tags based on Taskwarrior tags and projects but ignore "Description" field. Could you please advise if it's currently possible?

**Comment by lauft** [comment]
Currently, the hook script is not configurable. As a quick measure, replacing [line 48](https://github.com/GothenburgBitFactory/timewarrior/blob/446afd0818a97888c3fe049f442616a7a2829afd/ext/on-modify.timewarrior#L48) in the `on-modify.taskwarrior` script with
```
tags = []
```
should do the trick.
